internal/db/collection_define.go

```diff
@@ -120,15 +120,19 @@ func (db *DB) patchCollection(
 	if err != nil {
 		return err
 	}
-	txn := mustGetContextTxn(ctx)
-	cols, err := description.GetCollections(ctx, txn)
+	existingCols, err := db.getCollections(
+		ctx,
+		client.CollectionFetchOptions{IncludeInactive: immutable.Some(true)},
```

Comment:
todo: Please document why you are fetching the entire collection version history for the entire database.  I cannot tell why we would want inactive versions here, and it could be a very large amount of data.

Reply:
I'm fetching the entire collection version history because that's what it was previously doing. Only this time it includes the related schemas. It's needed because we are allowing patching of inactive collections as described in the tests you added for this.

Reply:
Ah, so it does, thanks!  A comment would still be handy here, but fair enough if you want to dismiss that as out of scope.